crates/uv/tests/pip_compile.rs

```diff
@@ -6922,11 +6922,11 @@ fn universal_transitive_disjoint_locals() -> Result<()> {
         # via torchvision
     sympy==1.12
         # via torch
-    torch==2.0.0+cpu
+    torch==2.0.0+cpu ; platform_machine != 'x86_64'
         # via
         #   -r requirements.in
         #   torchvision
-    torch==2.0.0+cu118
+    torch==2.0.0+cu118 ; platform_machine == 'x86_64'
```

Comment:
This looks like another legitimate bug fix, where previously we could install two different versions of `torch`.